client/signup/jetpack-connect/index.jsx

```diff
@@ -152,76 +128,108 @@ const JetpackConnectMain = React.createClass( {
-					</LoggedOutFormLinks>
+					{ this.renderSiteInput( status ) }
+					{ this.renderFooter() }
+				</div>
+			</Main>
+		);
+	},
+
+	renderInstallInstructions() {
+		return (
+			<Main className="jetpack-connect-wide">
+				<div className="jetpack-connect__install">
+					<ConnectHeader headerText={ this.translate( 'Ready for installation' ) }
+						subHeaderText={ this.translate( 'We\'ll need to send you to your site dashboard for a few manual steps' ) }
+						step={ 1 }
+						steps={ 3 } />
+					<div className="jetpack-connect__install-steps">
+						<JetpackInstallStep title={ this.translate( '1. Install Jetpack' ) }
+							text={ this.translate( 'You will be redirected to the Jetpack plugin page on your site\'s dashboard to install Jetpack. Click the blue install button.' ) }
+							example={ <JetpackExampleInstall /> } />
+						<JetpackInstallStep title={ this.translate( '2. Activate Jetpack' ) }
+							text={ this.translate( 'Once the plugin is installed, you\'ll need to click this tiny blue \'Activate\' link from your plugins list page.' ) }
+							example={ <JetpackExampleActivate /> } />
+						<JetpackInstallStep title={ this.translate( '3. Connect Jetpack' ) }
+							text={ this.translate( 'Once the plugin is activated you\'ll click this blue \'Connect\' button to complete the connection.' ) }
+							example={ <JetpackExampleConnect /> } />
+					</div>
+					<Button onClick={ this.installJetpack } primary>{ this.translate( 'Install Jetpack' ) }</Button>
+				</div>
+			</Main>
+		);
+	},
+
+	renderActivateInstructions() {
+		return (
+			<Main className="jetpack-connect-wide">
+				<div className="jetpack-connect__install">
+					<ConnectHeader headerText={ this.translate( 'Ready for installation' ) }
+						subHeaderText={ this.translate( 'We\'ll need to send you to your site dashboard for a few manual steps' ) }
+						step={ 1 }
+						steps={ 3 } />
+					<div className="jetpack-connect__install-steps">
+						<JetpackInstallStep title={ this.translate( '1. Activate Jetpack' ) }
+							text={ this.translate( 'You need to click this tiny blue \'Activate\' link from your plugins list page.' ) }
+							example={ <JetpackExampleActivate /> } />
+						<JetpackInstallStep title={ this.translate( '2. Connect Jetpack' ) }
+							text={ this.translate( 'Once the plugin is activated you\'ll click this blue \'Connect\' button to complete the connection.' ) }
+							example={ <JetpackExampleConnect /> } />
+					</div>
+					<Button onClick={ this.activateJetpack } primary>{ this.translate( 'Activate Jetpack' ) }</Button>
 				</div>
 			</Main>
 		);
+	},
+
+	render() {
+		const status = this.getStatus();
+		if ( status === 'notJetpack' && ! this.props.jetpackConnectSite.isDismissed ) {
+			return this.renderInstallInstructions();
+		}
+		if ( status === 'notActiveJetpack' && ! this.props.jetpackConnectSite.isDismissed ) {
+			return this.renderActivateInstructions();
+		}
+//		return this.renderActivateInstructions();
```

Comment:
To be removed once this is complete... @rickybanister, I let this here so it's easy to test both screens without having to introduce a real site, just uncomment whatever you need to test and that will be the default first screen in jetpack-connect